Renegade X Launcher/RenXWebLinks.cs

```diff
@@ -7,8 +7,8 @@
 
     public static class RenXWebLinks
     {
-        public static readonly char RENX_SERVER_SETTING_SPACE_SYMBOL = ';';
-        public const string RENX_ACTIVE_SERVER_JSON_URL = "http://serverlist.renegade-x.com/servers.jsp";
-        public const string RENX_BANNERS_JSON_URL = "https://static.renegade-x.com/launcher_data/banners";
+        public static readonly char RenxServerSettingSpaceSymbol = ';';

```

Comment:
I think we can actually remove this variable, since we use a JSON server list now instead of the Gemini based one